### PR TITLE
Use twitter gem for Streaming API

### PR DESCRIPTION
**Pull request by eagletmt**
twitter >= 5  has own Streaming API implementation independent of eventmachine.

### CODE REVIEW

File: lib/fluent/plugin/in_twitter.rb [hunk 1]
```diff
@@ -6,8 +6,8 @@ class TwitterInput < Fluent::Input
 
     config_param :consumer_key, :string
     config_param :consumer_secret, :string
-    config_param :oauth_token, :string
-    config_param :oauth_token_secret, :string
+    config_param :access_token, :string
+    config_param :access_token_secret, :string
```
Comment (eagletmt) [inline lib/fluent/plugin/in_twitter.rb]: Let's rename these configuration parameters for consistency with out_twitter plugin and twitter gem.

**Comment by y-ken** [comment]
Hi @eagletmt 

It seems very nice !! Thank you very much! :sunglasses: 
I'll check the behavior and the compatibility for present users.

**Comment by eagletmt** [comment]
any progress?

**Comment by okkez** [comment]
Any progress? @y-ken

**Comment by y-ken** [comment]
@okkez 
It has faced conflicts that should be resolved.
I need help to resolve this conflicts.

**Comment by okkez** [comment]
@eagletmt Do you have enough time to resolve this?
If no, I will resolve conflict in next month.

**Comment by y-ken** [comment]
@okkez Thank you very much.

**Comment by y-ken** [comment]
It has merged with PR https://github.com/y-ken/fluent-plugin-twitter/pull/28

**Comment by y-ken** [comment]
@eagletmt @okkez 
I'm sorry for late.
Thank you for enhancement for this plugin. 👍

**Pull request closed without merging**